frontend/src/routes/settings/connection/DatabaseConnectionForm.tsx

```diff
@@ -1,25 +1,12 @@
 import { z } from "zod";
-import {
```

Comment:
These changes seem to happen based on the prettier config, let me know if that's an issue

Reply:
The CI will complain yes. If you run `npx prettier . --write` it should use the projects config.
I did this already now though.